tracer/src/Datadog.Trace.ClrProfiler.Native/rejit_preprocessor.cpp

```diff
@@ -25,15 +25,23 @@ void RejitPreprocessor<RejitRequestDefinition>::ProcessTypeDefForRejit(const Rej
                                           std::vector<mdMethodDef>& vtMethodDefs)
 {
     auto target_method = GetTargetMethod(definition);
+    const bool wildcard_enabled = target_method.method_name == tracemethodintegration_wildcardmethodname;
```

Comment:
@GreenMatan The new code paths added occur when `target_method.method_name == "*"`. Since this is a shared code path, I wanted to get your feedback on this modification. Will this be okay for the debugger? Any changes you would propose?